modules/standard/Map.chpl

```diff
@@ -79,6 +79,14 @@ module Map {
     /* If `true`, this map will perform parallel safe operations. */
     param parSafe = false;
 
+    /* 
+       Fractional value that specifies how full this map can be 
+       before requesting additiotional memory. Default value is 0.5,
+       which means that the map will not resize until the map is more
+       than 50% full.
+    */
```

Comment:
1. Could you make it clear that the acceptable values are an open interval? i.e. it is (0,1) and not [0,1].

2. You can alternatively make `const resizeThreshold = 0.5`. chpldoc should render the default value (please check if you make that change). It'll be more consistent with the `parSafe` argument which also have a default value. If you drop that you can drop the default value in the doc text, but then, you give an example, which is nice. So, maybe you can keep it regardless?